rtris.py

```diff
@@ -128,11 +133,10 @@ def opt_version_info(opt, arg):
 	print(VERSION_INFO)
 	exit(0)
 def opt_update(opt, arg):
+	global update
 	if arg != None:
 		raise Exception("%s: too many arguments: 1" % (opt))
-	updater=Updater()
-	updater.update()
-	exit(0)
+	update=True
```

Comment:
I did this on purpose, for speed. If this updates right there, much less code has to run. I plan on doing this even more with arguments like -h or -V.